### PR TITLE
Add system.metrics as source for charts

### PR DESCRIPTION
**Pull request by kmatt**
This UNIONs system.events and system.metrics for additional sources of data. The term "metric" was used globally for both events and metrics.

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -24,6 +24,58 @@ type Graph struct {
 
 // AvailableGraphs is list of available graphs
 var AvailableGraphs = map[string]Graph{
+	"memory": {
+		graphTitle:    "ClickHouse memory",
+		graphCategory: "clickhouse",
+		graphInfo:     "Values received from ClickHouse system.events table",
```
Comment (flrnull) [inline config.go]: system.metrics )

**Review by flrnull — COMMENTED** [review]
It will be good if you add screenshot for example (but not necessary)

File: config.go (hunk 1 above)
Comment (kmatt) [reply config.go]: Once the charts populate enough I will tomorrow.

**Comment by kmatt** [comment]
Added new example of merge count chart.

**Comment by flrnull** [comment]
Yup, cool.
But what about typo in graphInfo? 
system.events —> system.metrics

**Comment by kmatt** [comment]
Fixed graphInfo typo. Note on my system (v1.1.54385) no cache events or metrics appear in the query. Not sure if that is a Clickhouse version or config issue.

**Pull request merged**